src/vs/base/parts/quickinput/browser/quickInput.ts

```diff
@@ -328,6 +352,13 @@ class QuickInput extends Disposable implements IQuickInput {
 		this.ui.ignoreFocusOut = this.ignoreFocusOut;
 		this.ui.setEnabled(this.enabled);
 		this.ui.setContextKey(this.contextKey);
+
+		const validationMessage = this.validationMessage || this.noValidationMessage;
+		if (this._lastValidationMessage !== validationMessage) {
+			this._lastValidationMessage = validationMessage;
+			dom.reset(this.ui.message, ...renderLabelWithIcons(escape(validationMessage)));
```

Comment:
When moving this from the subclasses to here, I saw that one sublcass ues `escape` and the other didn't. I opted to go with the `escape`, but I'm not 100% sure which is correct.

Reply:
`escape` seems to make sense in both cases. 👍